criteria/criteria.go

```diff
@@ -201,6 +202,21 @@ func Equals(left Expression, right Expression) Expression {
 	return reparent(&EqualsExpression{binaryExpression{expression{}, left, right}})
 }
 
+// SubstringExpression represents the equality operator
```

Comment:
Does it represent the `equality operator`?

Reply:
Fixed. fd55350a2ddb68d7a25f931ca7e53758c2527c7b